front/admin_ui/app.py

```diff
@@ -128,6 +136,52 @@ def auth(token):
                 auth_error: gr.update(value=f"❌ Unauthorized (user '{user['name']} is not a member of '{ADMIN_HF_ORGANIZATION}')")
             }
 
+    def call_obsolete_cache(token, delete):
+        headers = {"Authorization": f"Bearer {token}"}
+        obsolete_cache_endpoint = f"{DSS_ENDPOINT}/admin/obsolete-cache"
+        response = (
+            requests.delete(obsolete_cache_endpoint, headers=headers, timeout=240)
```

Comment:
it's a very big timeout. As it's an admin endpoint, I think it's OK to just set a large value instead of trying to paginate the results, for example (not worth it)